bids_prov/afni/afni_parser.py

```diff
@@ -0,0 +1,361 @@
+import argparse
```

Comment:
How much is this code same as the FSL parser? Could we factorize some of the code (to avoid code duplication and ease future updates)?

Reply:
It is the same skeleton, but there is some differences for each command (inside the for loop): 
 - section in fsl are taken into account
 - yet,  no parameters are extracted in fsl 
 -  in afni, there is a renaming step for entities , due to extra suffixes like `"tlrc"` 
 - preprocesing step to get clean command (multilines, drop useless commands ..) is different for afni and fsl
In any case, I just refactored function `args_consumed_list=get_entities(cmd_s, parameters)`  , taking account of consumed args while parsing them

Reply:
Ok thanks! Let's keep that in mind for future devs.